.github/workflows/vis-reg-tests.yml

```diff
@@ -2,20 +2,33 @@ name: Visual Regression Tests
 on: [push]
 
 jobs:
-  build:
+  design-system:
     timeout-minutes: 20
-
     runs-on: ubuntu-latest
-
     steps:
       - uses: actions/checkout@v3
       - uses: ./.github/actions/ci-setup
 
       - name: Build Storybook for Design System
-        run: pnpm storybook:ds:build
+        run: bash scripts/build-storybook.sh design-system
```

Comment:
would this not work?
`turbo run storybook:build --filter=@webstudio-is/design-system` 

If I understand correctly you created a custom bash script to just pass the filter argument?

Reply:
Yeah. Could use that way as well.